code/modules/modular_computers/computers/item/role_tablet_presets.dm

```diff
@@ -350,6 +350,8 @@
 
 /// Return true if our wearer is in a position to slip someone
```

Comment:
```suggestion
/// Returns whether the PDA can slip or not, if we have a wearer then check if they are in a position to slip someone.
```
Dunno if this is best but good to mention IF there's a wearer.

Reply:
sure